docs/development/extensions-core/avro.md

```diff
@@ -22,47 +22,43 @@ title: "Apache Avro"
   ~ under the License.
   -->
 
-## Avro extension
+This Apache Druid extension enables Druid to ingest and parse the Apache Avro data format. The Avro extension enables the following ingestion input methods:
```

Comment:
"extension enables Druid" ... "extension enables the following.." seems repetitive